src/evaluation_system/misc/config.py

```diff
@@ -363,4 +362,6 @@ def get_drs_config():
     )
     with open(drs_config, "r") as drs_file:
         _drs_config = toml.load(drs_file)
+    for key in _drs_config:
+        _drs_config[key].setdefault("root_path", _drs_config[key].get("root_dir", ""))
```

Comment:
here we are considering all `root_dir`'s values as `root_path` without giving any log to the user to make a bit of changes there. What do u think to add at least a warning log here?